UserControls/UserControls.vbproj

```diff
@@ -53,13 +53,13 @@
       <HintPath>..\packages\HEC.MVVMFramework.Base.1.0.0-alpha-2022.7.8.2331\lib\netstandard2.0\Base.dll</HintPath>
     </Reference>
     <Reference Include="fda-model, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\USACE.HEC.FDA.Model.1.1.0-alpha-2022-07-15-2223\lib\net48\fda-model.dll</HintPath>
+      <HintPath>..\packages\USACE.HEC.FDA.Model.1.1.0-alpha-2022-07-19-1451\lib\net48\fda-model.dll</HintPath>
     </Reference>
     <Reference Include="Model, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\HEC.MVVMFramework.Model.1.0.0-alpha-2022.7.8.2331\lib\netstandard2.0\Model.dll</HintPath>
     </Reference>
     <Reference Include="Statistics, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\USACE.HEC.FDA.Statistics.1.1.0-alpha-2022-07-15-2223\lib\net48\Statistics.dll</HintPath>
```

Comment:
Go ahead and drop this reference. This reference shouldn't exist at all. User controls doesn't need to know anything about statistics.

Reply:
also doesn't know anything about model. Drop that guy too.